fetch.bs

```diff
@@ -770,6 +770,9 @@ production as
 <ol>
  <li><p>Let <var>value</var> be <var>header</var>'s <a for=header>value</a>.
 
+ <li><p>If <var>value</var>'s <a for="byte sequence">length</a> is greater than 128, then return
+ false.
```

Comment:
Feels like this simple condition should go before the more complex conditions. I don't feel strongly about it though.